code/datums/ruins/lavaland.dm

```diff
@@ -150,6 +150,15 @@
 	name = "Blood-Drunk Miner (Hunter)"
 	suffix = "lavaland_surface_blooddrunk3.dmm"
 
+/datum/map_template/ruin/lavaland/blood_drunk_miner/random
+	name = "Blood-Drunk Miner (Random)"
+	suffix = null
+	always_place = TRUE
+
+/datum/map_template/ruin/lavaland/blood_drunk_miner/random/New()
+	suffix = pick("lavaland_surface_blooddrunk1.dmm", "lavaland_surface_blooddrunk2.dmm", "lavaland_surface_blooddrunk3.dmm")
+	return ..()
+
```

Comment:
Shouldn't you then remove the existing BDM ruins from this file?

Reply:
no, admins may want to spawn them.

Reply:
Makes sense